bin/runtime/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph-runtime"
-version = "0.3.0"
+version = "0.1.0"
```

Comment:
Going back with a version sounds very wrong when it's not a literal rollback. Could we bump all the versions instead?

This is a problem in a couple of places, but I'll only write it here, for discussion purposes.

Reply:
Definitely, an rollback error.